### ISSUE TITLE
Add item/fluid hatches for multiblocks that don't have them

### ISSUE THREAD

**Issue by cgdyusijhcgfdyuei** [dercodeKoenig/AdvancedRocketry#17]
**What do you think is missing or what do you think can be done better**
Some multiblocks (like the Electrolyzer), don't have item/fluid input/output hatches, not allowing them to process/output items/fluids. It would be appreciated if the structure requirements for these are modified so that any machine structure block (if there are any) can be substituted for an item/fluid input/output hatch.

**Comment by dercodeKoenig** [comment]
I dont understand this. the electrolyzer has fluid in and outputs. it can process water to oxygen and hydrogen.

**Comment by cgdyusijhcgfdyuei** [comment]
I guess I didn't word it correctly. [The Electrolyzer doesn't have item input & output hatches](https://imgur.com/a/94QAjgn). Other multiblocks (Cutting Machine, Lathe) don't have fluid hatches, that's what I meant when I said fluid input/output hatches.

**Comment by dercodeKoenig** [comment]
this is correct. it does no need item hatches. it electrolyses liquids. Do you want to make a custom recipe that makes items?

**Comment by cgdyusijhcgfdyuei** [comment]
Pretty much, that's why I made this suggestion

**Comment by dercodeKoenig** [comment]
Now I understand. yes I think i can add this
